pandas/_libs/algos.pyx

```diff
@@ -76,7 +76,7 @@ class NegInfinity(object):
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-cpdef ndarray[int64_t, ndim=1] unique_deltas(ndarray[int64_t] arr):
+cpdef ndarray[int64_t, ndim=1] unique_deltas(const int64_t[:] arr):
```

Comment:
does const provide any perf enhancements?

Reply:
I am not sure.

Reply:
ok thanks.